### PR TITLE
Fixed: CV alla Grid

### PR DESCRIPTION
**Pull request by SzabiSch**
Good evening Mrs Sissi, please review my NEW World App :) Thanks and Welcome in The Matrix, the black Cat

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,17 +30,20 @@ <h1 class="head-name">Szabi Sreiber</h1>
...
-        <h2>Profile</h2>
-        <p>
-          Hi everybody, I am Szabi Student of Coding Bootcamp Europe. I am
-          intresting long time for <strong>WEB-Development</strong>, and just
-          because I am here in sorty time learn much as possible
-        </p>
-      </article>
-
-      <article id="experience">
+      <div class="oben">
```
Comment (GagaSissi) [inline index.html:33]: Szabi, warum div?

**Review by GagaSissi — APPROVED** [review]
Please check if div is necessary.

**Comment by GagaSissi** [comment]
All fine.

File: index.html (hunk 1 above)
Comment (SzabiSch) [reply index.html:33]: just because :) but in this place it was necessary because of Grid

**Pull request closed without merging**